### ISSUE TITLE
More info on "Failed to load the tool-bar package" issues. Temporary solution for users.

### ISSUE THREAD

**Issue by thedonciuxx** [suda/tool-bar#324]
I am too getting the same exact error, but I want to give a bit more information than the other issues.

Not 100% sure, but it looks like tool-bar can't generate the dist/iconsets.css file.

In the [551d2ce8cda320be0b881f164139bcdc97cbe9bd commit](https://github.com/suda/tool-bar/commit/551d2ce8cda320be0b881f164139bcdc97cbe9bd), if I revert the change done to the styles/tool-bar.less file, the error disappears, but the tool-bar doesn't show up.

So I tested a cloned a few versions and compared commits.

For me up to commit `ab847968c66399f30328b2e9786248f32450d787` tool-bar worked.
Commit  `f6170fafcf59baba7083480cb6fb0638b388cd00` was suddenly when the tool-bar didn't even show up, but no errors.
And commit `551d2ce8cda320be0b881f164139bcdc97cbe9bd` was when the error appeared.

TL;DR
Something broke and doesn't load right. **If you want to get tool-bar working again** then you could try to downgrade your version by downloading the build from an older version like [v1.3.0](https://github.com/suda/tool-bar/releases/tag/v1.3.0) or the [lastest working commit](https://github.com/suda/tool-bar/archive/ab847968c66399f30328b2e9786248f32450d787.zip)

**Comment by suda** [comment]
Thank you for the workaround! I just released 1.4.1 which should fix the issue :)
